src/util.h

```diff
@@ -282,6 +282,17 @@ class ArgsManager
      * Check whether we know of this arg
      */
     bool IsArgKnown(const std::string& key, std::string& error);
+
+private:
+    /**
+     * Trim string by the pattern
+     */
+    std::string trimString(std::string str, const std::string& pattern);
```

Comment:
Is there a reason for this method to start with a lower case letter while all other start with upper case letters?

Reply:
No reason. So make it uppercase.